angular/src/services/bitwarden-icons.service.ts

```diff
@@ -0,0 +1,142 @@
+import { Injectable } from "@angular/core";
+
+// Update icon references if there is a change in the Bitwarden Icon Font
+const IconMap: any = {
```

Comment:
remove `any` from here. I believe we could make `IconMap` a static class and just reference it directly instead of having a helper service?

Reply:
Removed service and added map class.